content/en/docs/kubernetes/operator/troubleshooting/target-allocator.md

```diff
@@ -0,0 +1,488 @@
+metadata:
+  name: otelcol
+  namespace: opentelemetry
+spec:
+  mode: statefulset
+  targetAllocator:
+    enabled: true
+    serviceAccount: opentelemetry-targetallocator-sa
+    prometheusCR:
+      enabled: true
+      podMonitorSelector: {}
+      serviceMonitorSelector: {}
+  config:
+    receivers:
+      otlp:
+        protocols:
+          grpc: {}
+          http: {}
+      prometheus:
+        config:
+          scrape_configs:
+            - job_name: 'otel-collector'
+              scrape_interval: 10s
+              static_configs:
+                - targets: ['0.0.0.0:8888']
+
+    processors:
+      batch: {}
+
+    exporters:
+      logging:
+        verbosity: detailed
+
+    service:
+      pipelines:
+        traces:
+          receivers: [otlp]
+          processors: [batch]
+          exporters: [logging]
+        metrics:
+          receivers: [otlp, prometheus]
+          processors: []
+          exporters: [logging]
+        logs:
+          receivers: [otlp]
+          processors: [batch]
+          exporters: [logging]
+```
+
+First, set up a `port-forward` in Kubernetes, so that you can expose the Target
+Allocator service:
+
+```shell
+kubectl port-forward svc/otelcol-targetallocator -n opentelemetry 8080:80
+```
+
+Where `otelcol-targetallocator` is the value of `metadata.name` in your
+`OpenTelemetryCollector` CR concatenated with the `-targetallocator` suffix, and
+`opentelemetry` is the namespace to which the `OpenTelemetryCollector` CR is
+deployed.
+
+Next, get a list of jobs registered with the Target Allocator:
+
+```shell
+curl localhost:8080/jobs | jq
+```
+
+Your sample output should look like this:
+
+```json
+{
+  "serviceMonitor/opentelemetry/sm-example/1": {
+    "_link": "/jobs/serviceMonitor%2Fopentelemetry%2Fsm-example%2F1/targets"
+  },
+  "otel-collector": {
+    "_link": "/jobs/otel-collector/targets"
+  },
+  "serviceMonitor/opentelemetry/sm-example/0": {
+    "_link": "/jobs/serviceMonitor%2Fopentelemetry%2Fsm-example%2F0/targets"
+  },
+  "serviceMonitor/opentelemetry/sm-example/2": {
+    "_link": "/jobs/serviceMonitor%2Fopentelemetry%2Fsm-example%2F2/targets"
+  },
+  "podMonitor/opentelemetry/pm-example/0": {
+    "_link": "/jobs/podMonitor%2Fopentelemetry%2Fpm-example%2F0/targets"
+  }
+}
+```
+
+Where `serviceMonitor/opentelemetry/sm-example/0` represents one of the
+`Service` ports that the `ServiceMonitor`picked up:
+
+- `opentelemetry` is the namespace in which the `ServiceMonitor` resource
+  resides.
+- `sm-example` is the name of the `ServiceMonitor`.
+- `0` is one of the port endpoints matched between the `ServiceMonitor` and the
+  `Service`.
+
+Similarly, the `PodMonitor`, shows up as `podMonitor/opentelemetry/pm-example/0`
+in the `curl` output.
+
+This is good news, because it tells us that the service discovery is working!
+
+You might also be wondering about the `otel-collector` entry. The Target
+Allocator automatically scrapes metrics from the OTel Collector as well.
```

Comment:
this entry is because [the config provided](https://github.com/open-telemetry/opentelemetry.io/pull/4708/files#diff-1dbe631d16dc9eb3b4e3af1e41c754015d43209646e535c073e124e5c4a7e2e9R100-R103) has the otel-collector self scrape set, the TA wouldn't have it if not for that block

Reply:
Thanks for clarifying! Added.